### ISSUE TITLE
Test that tabs are correctly hidden

### ISSUE THREAD

**Issue by djbuckley** [manchesergit/material-ui#201]
Add tests to ensure that : 
A Tab that is not currently active should be set to hidden, whilst the active one is not (aria-hidden)
